babel.config.js

```diff
@@ -1,4 +1,11 @@
 // babel.config.js
 module.exports = {
-  presets: ['@babel/preset-env', '@babel/preset-react'],
+  presets: [
+    "@babel/preset-env",
+    "@babel/preset-react"
+  ],
+  plugins: [
+    "@babel/transform-runtime",
+    "@babel/plugin-proposal-class-properties"
+  ]
```

Comment:
nit: needs trailing comma on lines 5, 9, and 10

Reply:
Done.